### DISCUSSION TITLE
Style on an entity with 4 states

### DISCUSSION THREAD

**Discussion by alexcool83** [Clooos/Bubble-Card#822] [Questions about config, custom styles and templates]
I have an entity that I want to change color with bubble card, which is not on-off, but has four states: alarm, ready, fault, short-circuit. 
I tried setting this in the style part of bubble-card but it always takes last color. The states are regularly detected by a test template that I made in home assistant.

```
.bubble-button-card-container {
    opacity: 1 !important;
    background-color: ${state === 'allarme' ? 'red' : state === 'guasto' ? 'orange' : state === 'allarme' ? 'red' : 'grey'} !important;
  } 
```

**Comment by MrBearPresident** [comment]
This can be done similar to #929. 

Also important is to know what the real state is and not the translated state. 
For this go to the developer tools (from home assistant) and search for your entity to check what state is reported.
